tests/test_protocol.py

```diff
@@ -1543,3 +1543,42 @@ def test_explicit_permanent_key(
             assert signed_keys == r['ssk'] + responder_key.pk
             yield from responder.close()
             yield from server.wait_connections_closed()
+
+    @pytest.mark.asyncio
+    def test_initiator_disconnected(
+            self, server, client_factory,
+    ):
+        # Client handshakes
+        initiator, i = yield from client_factory(initiator_handshake=True)
+        responder1, r = yield from client_factory(responder_handshake=True)
+        responder2, r = yield from client_factory(responder_handshake=True)
+
+        # Disconnect initiator
+        yield from initiator.close()
+
+        # Expect 'disconnected' msgs sent to all responders
+        msg1 = (yield from responder1.recv())[0]
```

Comment:
Nit: I prefer `msg1, *_ = ...` over `msg1 = ...[0]`

Reply:
Ah, I was looking for destructuring syntax like that :)